rcongui/src/components/PlayerInfo/Player.css

```diff
@@ -0,0 +1,118 @@
+.player-info {
```

Comment:
With material UI we're supposed to do stuff with this pattern: https://material-ui.com/styles/basics/

I'm surprised you need that much css. A lot is handled out of the box by materiali UI.

for example all this:
```
    display: grid;
    justify-content: center;
    align-content: center;
 ```
 is 
 
 ```
 <Grid container justifyContent="center" ... >
 ```